main.go

```diff
@@ -398,17 +396,22 @@ func getSeedFromFactorio(rconClient *rcon.Client) string {
 	return seed
 }
 
-func updatePlayerCount(rconClient *rcon.Client) {
+func getPlayersOnlineFromFactorio(rconClient *rcon.Client) (int, error) {
 	msg, err := rconClient.Execute("/players online count")
 	if err != nil {
 		log.WithFields(logrus.Fields{"err": err}).Error("Could not get player count from Factorio")
-		return
+		return 0, err
 	}
-	playersOnline, err = strconv.Atoi(strings.Split(strings.Split(msg, "(")[1], ")")[0])
+	playersOnline, err := strconv.Atoi(strings.Split(strings.Split(msg, "(")[1], ")")[0])
```

Comment:
I don't think this works as you now redeclare the `playersOnline`, let me confirm with a test